src/System.Console/ref/System.Console.cs

```diff
@@ -36,8 +36,11 @@ public static partial class Console
         public static void MoveBufferArea(int sourceLeft, int sourceTop, int sourceWidth, int sourceHeight, int targetLeft, int targetTop, char sourceChar, ConsoleColor sourceForeColor, ConsoleColor sourceBackColor) { }
         public static bool NumberLock { get { return default(bool); }}
```

Comment:
I pushed a commit to update the contract version.  Can you please check to see if I did it correctly?  Without a .cs file in the 4.0.0 folder, I don't understand how it knows what the previous version of the contract should look like.